util/osuser/user.go

```diff
@@ -129,6 +123,14 @@ func userLookupGetent(usernameOrUID string, std lookupStd) (*user.User, string,
 	for len(f) < 7 {
 		f = append(f, "")
 	}
+	var mandatoryFields = []int{0, 2, 3, 5}
+	for _, v := range mandatoryFields {
+		if f[v] == "" {
+			log.Printf("getent for user %q returned invalid output: %s", usernameOrUID, out)
```

Comment:
```suggestion
			log.Printf("getent for user %q returned invalid output: %q", usernameOrUID, out)
```

Reply:
I'll roll this change up once we resolve the conversation above.